pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -269,8 +286,7 @@ private void createOrUpdateRateLimiter(String tableNameWithType, ExternalView br
     double overallRate = quotaConfig.getMaxQPS();
 
     // Get stat from property store
-    String tableConfigPath = constructTableConfigPath(tableNameWithType);
-    Stat stat = _propertyStore.getStat(tableConfigPath, AccessOption.PERSISTENT);
+    Stat stat = _propertyStore.getStat(physicalOrLogicalTableConfigPath, AccessOption.PERSISTENT);
```

Comment:
Can `Stat` be passed in as a parameter or should it be got only after the above checks ?

Reply:
We can pass in `Stat` but I kept it as `physicalOrLogicalTableConfigPath` if it can used for something more in the future.

Reply:
The callers are also complicated. My suggestion is complicate when required in the future.

Reply:
More complicated than before :)